nix/grpc-pin/grpc.nix

```diff
@@ -0,0 +1,108 @@
+# Copy-paste from nixpkgs “release-22.05” branch
```

Comment:
Do we still need to pin `grpc.nix` if `nixpkgs.nix` now uses `release-22.05`?

Same question for the other `grpc`-related pins

Reply:
@Gabriella439 technically no but the library is often used with different nixpkgs pins, not one provided by gRPC-haskell. So gRPC version can be different, thus there can be breaking changes. This feels more reliable to provide fixed pin of gRPC version this library specifically is built for.